tools/delete_imaging_upload.pl

```diff
@@ -0,0 +1,760 @@
+#!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+delete_mri_upload.pl -- Delete everything that was produced by the imaging pipeline for a given set of imaging uploads
+
+=head1 SYNOPSIS
+
+perl delete_mri_upload.pl [-profile file] [-ignore] [-nobackup] [-uploadID lis_of_uploadIDs]
+
+Available options are:
+
+-profile     : name of the config file in C<../dicom-archive/.loris_mri> (defaults to C<prod>).
+
+-ignore      : when performing the file backup, ignore files that do not exist or are not readable
+               (default is to abort if such a file is found). This option is ignored if C<-n> is used.
+               
+-nobackup    : do not backup the files produced by the imaging pipeline for the upload(s) passed on
+               the command line (default is to perform a backup).
+               
+-uploadID    : comma-separated list of upload IDs (in table C<mri_upload>) to delete.
+
+
+=head1 DESCRIPTION
+
+This program deletes all the files and database records produced by the imaging pipeline for a given set
+of imaging uploads that have the same C<TarchiveID> in table C<mri_upload>. The script will issue and error
+message and exit if multiple upload IDs are passed on the command line and they do not all have the 
+same C<TarchiveID>. The script will remove the records associated to the imaging upload whose IDs are passed
+on the command line from the following tables: C<notification_spool>, C<tarchive_series>
+C<tarchive_files>, C<files_intermediary>, C<parameter_file>, C<files>, C<mri_violated_scans>
+C<mri_violations_log>, C<MRICandidateErrors>, C<mri_upload> and C<tarchive>. It will also delete from
+the file system the files that are associated to the upload and are listed in tables C<files>
+C<files_intermediary> and C<parameter_file>, along with the archive itself, whose path is stored in 
+table C<tarchive>. The script will abort and will not delete anything if there is QC information
```

Comment:
would also add mri_violations_log, MRICandidateErrors and mri_protocol_violated_scans in the sentence "It will also delete from the file system the files that are associated to the upload and are listed in tables..."

Reply:
Ok